docs/orchestration/concepts/api_keys.md

```diff
@@ -1,20 +1,25 @@
 # API Keys <Badge text="Cloud"/>
 
-API keys are a central component for interacting with the Prefect Cloud API.  They encapsulate who you are as a user of the system along with your memberships to your various tenants.  API keys can also be generated for Service Accounts, allowing for keys with customizable permission sets.  Ultimately, all clients that interact with the Prefect Cloud API must provide an API key as a Bearer Token included in the request header.
+API Keys are how clients authenticate with the Prefect Cloud API.  They encapsulate the identity of a User or a Service Account.  Ultimately, all clients that interact with the Prefect Cloud API must provide an API Key as a Bearer Token included in the request header.
 
+For more information on how to use these keys to interact with the GraphQL API, go [here](api.html).
 ### User API Keys
 
-User-based API keys function as personal access keys.  When creating an API key, you can configure the following attributes:
+Users can generate API Keys to serve as personal access keys.  When creating an API key, you can configure the following attributes:
```

Comment:
Introducing the personal access key here is confusing

```suggestion
Users can generate API Keys to interact with the API with their personal permissions.  When creating an API key, you can configure the following attributes:
```

Reply:
I was avoiding changing that particular language but I agree